Makefile

```diff
@@ -8,26 +8,29 @@ else
 endif
 
 # Docker configurations
+export ARCH := $(shell uname -m)
```

Comment:
You may want to change this to allow overriding, and to normalize the value across platforms:

export ARCH ?= $(shell hzn architecture)